rstudio/rstudio
# Update Zoom Source to include columns

### Intent

Fixes #7801 

Enhances the `Zoom Source` command so it zooms in on the most recently accessed Source pane - this may be a Source Column.

### Approach

I updated `onLayoutZoomSource` to specify the active source column rather than the defaulting to the main column. While researching this PR I realized that our zooming animation code was unnecessary as `MainSplitPanel` has an animate function we can use to get the same effect so I removed a fair amount of the existing code as well.

### QA Notes

Open several source columns and move focus between them, then select 'Zoom Source'. The most recently selected column should be zoomed in on.
